Audio/AudioEngine.cpp

```diff
@@ -1450,15 +1458,15 @@ X3DAUDIO_HANDLE& AudioEngine::Get3DHandle() const noexcept
 
 
 // Static methods.
```

Comment:
The code here down has been rearranged a fair bit, so the diff is really hard to see. You can view the file and look at it directly.